app/talk/discussion-preview.cjsx

```diff
@@ -15,21 +15,25 @@ module?.exports = React.createClass
   propTypes:
     discussion: React.PropTypes.object
 
+  contextTypes:
+    geordi: React.PropTypes.object
+
   discussionLink: ->
+    logClick = @context.geordi?.makeHandler? 'view-discussion'
```

Comment:
Why is this done with a different approach than some of the other files below? (no `makeHandler` in some of the others)